app/Transformers/BeatmapCompactTransformer.php

```diff
@@ -17,6 +17,10 @@ class BeatmapCompactTransformer extends TransformerAbstract
         'max_combo',
     ];
 
+    protected $defaultIncludes = [
```

Comment:
One thing I'm not sure about is whether this should be here or in `BeatmapTransformer`.

Reply:
im not very sure but if this compacted beatmap should have checksum as default, i think it should be in transform function (and remove related code)

Reply:
Good point, I've left it here for now so it doesn't break any instances that depend on the `checksum` include, but if we actually decide on keeping it here it should be removed, yes.